File: Factorial Fibonacci GCD - Python/fibonacci.py
def fibo(n):
    if n==1:
        return 0
    elif n==2:
        return 1
    else:
        return fibo(n-1)+fibo(n-2)
n=int(input("Enter the value of n: "))
if n<0:
    print("Enter a number greater than 0")
else:
    print("The fibonacci number is "+str(fibo(n)))

File: Factorial Fibonacci in Golang/fibonacci.go
// go run fibonacci.go

package main

import (
	"fmt"
)

func FibonacciRecursion(n int) int {
    if n <= 1 {
        return n
    }
    return FibonacciRecursion(n-1) + FibonacciRecursion(n-2)
}
func main() {
    fmt.Println("Enter the number:") 
	var n int
	fmt.Scanln(&n)
	fmt.Println("Answer: ",FibonacciRecursion(n))
}
